securedrop/i18n.py

```diff
@@ -34,8 +36,18 @@ class LocaleNotFound(Exception):
     """Raised when the desired locale is not in the translations directory"""
 
 
-def setup_app(app):
+def setup_app(app, translation_dirs=None):
     global babel
+
+    if translation_dirs is None:
+            translation_dirs = \
+                    path.join(path.dirname(path.realpath(__file__)),
+                              'translations')
+
+    # `babel.translation_directories` is a nightmare
```

Comment:
You'd have to ask @dachary  about that one. I don't know why, but this had to happen.

Reply:
It took me forever to figure out that the translation_dirs has to be set as follows. This was because originally it was hardcoded. And it was semi-hacked into being configurable but ... not properly documented. On top of that the code dealing with this is difficult to read. In any case the nightmare is about **finding** the proper method. I did not mean to say that this method of configuring the translation dirs is a nightmare: it works very well ;-)